@here/generator-harp.gl/test/test.js

```diff
@@ -17,34 +13,35 @@ const {
 
 */
 describe("harp.gl:app", function () {
-
     this.timeout(300000);
 
     let helperDirectory;
 
     before(async function () {
-        helperDirectory = await yeomanTestHelpers.run(path.join(__dirname, '../generators/app'))
+        helperDirectory = await yeomanTestHelpers
+            .run(path.join(__dirname, "../generators/app"))
             .inTmpDir(function (dir) {
                 if (process.env.USE_NPMRC) {
                     const targetPath = path.join(dir, ".npmrc");
                     fs.copyFileSync(process.env.USE_NPMRC, targetPath);
                 }
             })
             .withPrompts({
-                'access_token': 'test_token'
+                language: "typescript",
+                apikey: "test_token"
             });
 
-        assert.isString(helperDirectory);
+        assert.isString(helperDirectory.cwd);
```

Comment:
`cwd` is not public according to https://yeoman.github.io/yeoman-test/YeomanTest.html, we might need to use a separate variable.
Can we assume that the current cwd is `path.join(__dirname, "../generators/app")` ?

Reply:
Good point, yes I guess that makes sense.

Actually checking the `isString` is anyway some what ambiguous, I will check that the run result is not undefined.

Reply:
I see the code: https://yeoman.github.io/yeoman-test/run-result.js.html

And there is a `cwd`, so whilst not part of the documentation, it is part of the public API, at least according to the code.

We call `inTmpDir`, which gives us some random tmp directory, so it is different that the path.join you mentioned and not so easy to get the `cwd`, at least I don't see a method to return it.